libexec/kvm-cluster-up/user-data

```diff
@@ -4,6 +4,14 @@ growpart:
   mode: auto
   devices: ['/']
 
+users:
+  - name: @@KUP_CLOUD_USERNAME@@
+    shell: /bin/bash
```

Comment:
"/bin/bash" won't work on cirros (and possibly alpine images).

Reply:
So, it seems to me that the stable points would be to either add a variable for the shell, or clearly indicate what the legal values are/enforce them.

Reply:
> add a variable for the shell,

This was the bit I was trying to avoid. It'll never end.

As things stand today you could try a cloud-based alpine image (you'll need to specify KUP_USERNAME=alpine) without requiring a modification to this package. I was trying to avoid policy as much as is possible. Supply two or three key things, the image will provision/boot and you should be good to go without any other changes. (Reality may be different.) But I believe I've had success as things stand with cirros, alpine, fedora, debian, rhel, centos -- but they do need to be cloud-based images.

If you're coming from a kickstart POV then you may find this package at odds with that.